core/src/main/java/org/elasticsearch/action/search/SearchScrollQueryAndFetchAsyncAction.java

```diff
@@ -170,7 +171,7 @@ private void finishHim() {
     private void innerFinishHim() throws Exception {
         ScoreDoc[] sortedShardList = searchPhaseController.sortDocs(true, queryFetchResults);
         final InternalSearchResponse internalResponse = searchPhaseController.merge(sortedShardList, queryFetchResults,
-            queryFetchResults);
+                queryFetchResults, null);
```

Comment:
The problem is that this class has a `SearchScrollRequest` rather than a SearchRequest so I'm not sure where we would get the SearchRequest from?